tests/tests_pytorch/strategies/test_fsdp.py

```diff
@@ -839,21 +839,27 @@ def test_fsdp_save_checkpoint_path_exists(shutil_mock, torch_save_mock, __, ___,
 
     strategy = FSDPStrategy(state_dict_type="sharded")
 
+    save_mock = mock.patch(
+        "torch.distributed.checkpoint.save"
+        if _TORCH_GREATER_EQUAL_2_2 else "torch.distributed.checkpoint.save_state_dict")
```

Comment:
let's get back any formatter

Reply:
I don't know why black was removed. Yes, it would be good to have a formatted again